lib/jsdom/living/events/Event-impl.js

```diff
@@ -22,6 +22,7 @@ class EventImpl {
     }
 
     this.target = null;
+    this.srcElement = null;
```

Comment:
Seems a bit better to just return `this.target`. That way it matches the spec and we don't have to worry about keeping them in sync.